treenode/models.py

```diff
@@ -391,9 +390,15 @@ def __get_node_order_str(self):
         alphabetical_val = slugify(str(self))
         alphabetical_key = alphabetical_val.ljust(priority_len, str('z'))
         alphabetical_key = alphabetical_key[0:priority_len]
-        pk_val = min(self.pk, priority_max)
+        if isinstance(self.pk, int):
+            pk_val = min(self.pk, priority_max)
+        elif isinstance(self.pk, uuid.UUID):
+            pk_val = self.pk.int
+            pk_val = int(str(pk_val)[:priority_len])
+        else:
+            pk_val = 0
```

Comment:
The latest condition is uncovered by tests, could you fix it?

Reply:
I wouldn't know how to test it. I thought about removing the else completely. What do you think?

Reply:
With this PR the library handles `uuid` and `int` as primary keys, this block could just check for `uuid` in the first condition and fallback to the `int` check by default in the else block.